apps/individuals/api/serializers.py

```diff
@@ -24,33 +30,198 @@ class Meta:
+                  'current_employment', 'contact_details']
+    
+    # Get the primary address    
+    def get_address(self, obj):
+        primary_address = obj.addresses.filter(is_primary=True).first()
+        if primary_address:
+            return AddressSerializer(primary_address).data
+        # fallback: return first address if no primary is set
+        latest_address = obj.addresses.order_by('id').first()
+        if latest_address:
+            return AddressSerializer(latest_address).data
+        return None
+
+    
+    def get_current_employment(self, obj):
+        current_employment = obj.employment_details.filter(is_current=True).first()
+        if current_employment:
+            return EmploymentDetailSerializer(current_employment).data
+        return None
+
+    
+class IndividualCreateSerializer(serializers.ModelSerializer):
+    addresses = AddressSerializer(many=True, required=True)
+    employment_details = EmploymentDetailSerializer(many=True, required=False)
+    next_of_kin = NextOfKinSerializer(many=True, required=False)
+    contact_details = ContactDetailsSerializer(many=True, required=False)
+    notes = NoteSerializer(many=True, required=False)
+    documents = DocumentSerializer(many=True, required=False)
+    class Meta:
+        model = Individual
+        fields = [
+            'id','first_name', 'last_name', 'date_of_birth', 'gender','marital_status',
+            'identification_type', 'identification_number','contact_details',
+            'addresses', 'employment_details', 'next_of_kin', 'documents','notes'
+        ]
+
+    def validate(self, data):
+        identification_number = data.get('identification_number')
+        if Individual.objects.filter(identification_number__iexact=identification_number).exists():
+            raise serializers.ValidationError({"identification_number": "This identification number is already registered."})
+        return data
+
+    @transaction.atomic
+    def create(self, validated_data):
+        address_data = validated_data.pop('addresses', [])
+        employment_data = validated_data.pop('employment_details', [])
+        kin_data = validated_data.pop('next_of_kin', [])
+        contact_data= validated_data.pop('contact_details', [])
+        documents_data = validated_data.pop('documents',[])
+        notes_data = validated_data.pop('notes', [])
+        user = self.context.get('user')
+
+        individual = Individual.objects.create(**validated_data)
+
+        # Create addresses
+        for address_data in address_data:
+            Address.objects.create(
+                user=user,
+                content_object=individual,
+                **address_data,
+                is_primary=True
+            )
+
+        for contact in contact_data:
+            IndividualContactDetail.objects.create(
+                user=user,
+                individual=individual,
+                **contact
```

Comment:
if address is provided, you can either process the number to start with country code or add a property method that returns a valid phone number country specific eg 2637** for zimbabwean numbers. add such as helper methods which can be used across the project


```suggestion
import re

def validate_zimbabwean_phone(phone):
    # Step 1: Clean input
    cleaned = re.sub(r'[+\s-]', '', phone)

    # Step 2: Validate
    if cleaned.startswith('07') and len(cleaned) == 10:
        if cleaned[2] in ['7', '1', '8', '3']:
            return True
    elif cleaned.startswith('2637') and len(cleaned) == 12:
        if cleaned[4] in ['7', '1', '8', '3']:
            return True
    return False
```